starter-code/views/celebrities/edit-view.ejs

```diff
@@ -0,0 +1,20 @@
+<h2>Edit Celebrity</h2>
+
+<form class="" action="/celebrities/<% celebrity._id %>" method="post">
```

Comment:
Your whole edit function broke because you forgot the `=` in your `< celebrity._id`